Resources/public/js/models/ez-usermodel.js

```diff
@@ -36,17 +36,46 @@ YUI.add('ez-usermodel', function (Y) {
             var api = options.api;
 
             if ( action === 'read' ) {
-                api.getUserService().loadUser(
-                    this.get('id'), callback
-                );
+                api.getUserService().loadUser(this.get('id'), function (err, response) {
+                    var contentType;
+
+                    if ( err ) {
+                        return callback(err, response);
+                    }
+
+                    contentType = new Y.eZ.ContentType({id: response.document.User.ContentType._href});
+                    contentType.load({api: api}, function (error) {
+                        response.document._contentType = contentType;
+                        callback(error, response);
+                    });
+                });
             } else {
                 callback("Only read operation is supported at the moment");
             }
+        },
+
+        /**
+         * Set the avatar to the user item.
+         *
+         * @method _parseStruct
+         * @param {Object} the user item
+         * @param {Object} the response document
+         */
+        _parseStruct: function (struct, responseDoc) {
+                var attrs, imageField,
+                    imageIdentifiers = responseDoc._contentType.getFieldDefinitionIdentifiers('ezimage');
+
+            attrs = this.constructor.superclass._parseStruct.call(this, struct);
+            imageField = struct.Version.Fields.field.filter(function (field) {
+                return field.fieldDefinitionIdentifier == imageIdentifiers[0];
+            });
+            attrs.avatar = imageField.length ? imageField[0].fieldValue : null;
+            return attrs;
         }
     }, {
         REST_STRUCT_ROOT: "User",
         ATTRS_REST_MAP: [
-            'email', 'login', 'enabled', 'name',
+            'email', 'login', 'enabled', 'name', 'avatar',
```

Comment:
you should not add `avatar` there. the `ATTRS_REST_MAP` list the attributes that can filled by simply reading the REST response. `avatar` is not in that category since you build its value in your custom `_parseStruct` method.